src/mmw/apps/bigcz/utils.py

```diff
@@ -8,6 +8,24 @@
 from rest_framework.exceptions import APIException
 import dateutil.parser
 
+from django.db import connection
+from geopandas import GeoDataFrame
+
+
+def get_huc_info(geom):
+    op = '@'
```

Comment:
The reason I'm using `@` and `&&` is because of https://github.com/BiG-CZ/BiG-CZ-Portal/issues/8#issuecomment-379899534. 

> I think it's possible that we could identify all the HUC12s that overlap with a WikiWatershed web app user's Area of Interest, fetch those, then do spatial cropping/filtering on our side similar to what we've done for our our WDC spatial searches.

I am using the huc12s to do the query rather than the bbox.

Reply:
> I am using the huc12s to do the query rather than the bbox.

Yes, and it's really neat that the API supports this! The bbox I'm talking about is not sent in the request, but the one used in the query.

* [`&&`](https://postgis.net/docs/RT_Raster_Intersect.html): Returns TRUE if A's **bounding box** intersects B's **bounding box**.
* [`ST_Intersects`](https://postgis.net/docs/RT_ST_Intersects.html): Return true if raster rastA **spatially intersects** raster rastB.

Using `&&` matches more HUC-12s than necessary since the bounding box of each HUC-12 is larger than the actual shape. This leads getting a greater than necessary set of HUC-12s, fetching more than necessary data from the API. By using `ST_Intersects` we ensure we get only the HUC-12s that actually overlap with the area of interest, and not those whose bounding box overlaps but the actual shape does not.

Does that make sense?

Reply:
That makes sense. Thanks.